app/views/application/notes/first-time.njk

```diff
@@ -23,9 +23,9 @@
         <h1 class="govuk-heading-l app-heading-l--inverte">Using notes</h1>
 
         <p class="govuk-body app-body--inverte">
-          When you add a note to an application, it will be visible to you and colleagues with access to applications for this course.<p>
+          When you add a note to an application, it will only be visible to you and colleagues with access to applications for this course.<p>
```

Comment:
```suggestion
          When you add a note to an application, it‘ll only be visible to you and colleagues with access to applications for this course.<p>
```

Reply:
This one doesn't work – it looks weird in the context of such a formal bit of reassurance :)